force-app/main/default/classes/Homework.cls

```diff
@@ -101,6 +120,7 @@ public with sharing class Homework
         // assign 'value' raised to the power of 'power' to result
         // e.g. if value is 10 and power is 2, result should be 100
         // do not use a loop!
+        result = value^power;
```

Comment:
The `^` operator isn't power of in Salesforce, it is bitwise XOR which won't give the answer you want. You'll need to find a different way to do this.